rtv/main.py

```diff
@@ -34,9 +36,11 @@ def main():
         formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('-s', dest='subreddit', help='subreddit name')
     parser.add_argument('-l', dest='link', help='full link to a submission')
+    parser.add_argument('-a', dest='force_ascii',
```

Comment:
What are your thoughts about changing this from ``-a`` to just ``--force_ascii``? I don't foresee this flag being used very often, and it might be good to reserve ``-a`` for something more general.